dhall-docs/src/Dhall/Docs.hs

```diff
@@ -150,7 +166,7 @@ saveHtml
     -> String                   -- ^ Package name
     -> (Path Abs File, Header)  -- ^ (Input file, Parsed header)
```

Comment:
(Similar nit regarding tuple vs separate params. Maybe you could instead use `uncurry` on the call site.)